### ISSUE TITLE
Game crashes when attempting to load a save that has been has blueprints pasted into it

### ISSUE THREAD

**Issue by caesarwease** [AnthorNet/SC-InteractiveMap#355]
# Issue
Game crashes when I attempt to load a save that has been has blueprints pasted into it. It does this with blueprints from one of my own functioning worlds and also a megaprint from Update 8

## Replication
Just try to load one of the saves I uploaded. I can't attach everything as the file size is too big.

### Actual Result
The game goes to the loading screen for a while then crashes with the following message:

An Unreal process has crashed: UE-FactoryGame

Version: 368883, IsEditor: No, IsPerforceBuild: No, BuildConfiguration: Shipping, Launcher: Steam, NetMode: Standalone, IsUsingMods: No, IsSaveGameEdited: Yes

Unhandled Exception: EXCEPTION_ACCESS_VIOLATION reading address 0x0000000000000210

FactoryGameSteam_FactoryGame_Win64_Shipping!AFGBuildableDroneStation::GetDroneDockingStartLocation() [C:\BuildAgent\work\b731a33f2a691e17\UE4\FactoryGame\Source\FactoryGame\Private\Buildables\FGBuildableDroneStation.cpp:626]
FactoryGameSteam_FactoryGame_Win64_Shipping!FDroneAction_DockingSequence::Begin() [C:\BuildAgent\work\b731a33f2a691e17\UE4\FactoryGame\Source\FactoryGame\Private\FGDroneVehicle.cpp:571]
FactoryGameSteam_FactoryGame_Win64_Shipping!AFGDroneVehicle::PostLoadGame_Implementation() [C:\BuildAgent\work\b731a33f2a691e17\UE4\FactoryGame\Source\FactoryGame\Private\FGDroneVehicle.cpp:1069]
FactoryGameSteam_FactoryGame_Win64_Shipping!IFGSaveInterface::execPostLoadGame() [C:\BuildAgent\work\b731a33f2a691e17\UE4\FactoryGame\Intermediate\Build\Win64\FactoryGameSteam\Inc\FactoryGame\UHT\FGSaveInterface.gen.cpp:48]
FactoryGameSteam_CoreUObject_Win64_Shipping!UFunction::Invoke() [C:\BuildAgent\work\b731a33f2a691e17\UE4\Engine\Source\Runtime\CoreUObject\Private\UObject\Class.cpp:6666]
FactoryGameSteam_CoreUObject_Win64_Shipping!UObject::ProcessEvent() [C:\BuildAgent\work\b731a33f2a691e17\UE4\Engine\Source\Runtime\CoreUObject\Private\UObject\ScriptCore.cpp:2161]
FactoryGameSteam_Engine_Win64_Shipping!AActor::ProcessEvent() [C:\BuildAgent\work\b731a33f2a691e17\UE4\Engine\Source\Runtime\Engine\Private\Actor.cpp:1175]
FactoryGameSteam_FactoryGame_Win64_Shipping!IFGSaveInterface::Execute_PostLoadGame() [C:\BuildAgent\work\b731a33f2a691e17\UE4\FactoryGame\Intermediate\Build\Win64\FactoryGameSteam\Inc\FactoryGame\UHT\FGSaveInterface.gen.cpp:559]
FactoryGameSteam_FactoryGame_Win64_Shipping!UFGSaveSession::DeserializePersistentAndRuntimeState() [C:\BuildAgent\work\b731a33f2a691e17\UE4\FactoryGame\Source\FactoryGame\Private\FGSaveSession.cpp:963]
FactoryGameSteam_FactoryGame_Win64_Shipping!UFGSaveSession::SerializeLoadedObjects() [C:\BuildAgent\work\b731a33f2a691e17\UE4\FactoryGame\Source\FactoryGame\Private\FGSaveSession.cpp:2566]
FactoryGameSteam_FactoryGame_Win64_Shipping!UFGSaveSession::LoadCompressedFileFromDisk() [C:\BuildAgent\work\b731a33f2a691e17\UE4\FactoryGame\Source\FactoryGame\Private\FGSaveSession.cpp:2149]
FactoryGameSteam_FactoryGame_Win64_Shipping!UFGSaveSession::LoadGame() [C:\BuildAgent\work\b731a33f2a691e17\UE4\FactoryGame\Source\FactoryGame\Private\FGSaveSession.cpp:2006]
FactoryGameSteam_FactoryGame_Win64_Shipping!AFGGameMode::InitGameState() [C:\BuildAgent\work\b731a33f2a691e17\UE4\FactoryGame\Source\FactoryGame\Private\FGGameMode.cpp:530]
FactoryGameSteam_Engine_Win64_Shipping!AGameModeBase::PreInitializeComponents() [C:\BuildAgent\work\b731a33f2a691e17\UE4\Engine\Source\Runtime\Engine\Private\GameModeBase.cpp:135]
FactoryGameSteam_Engine_Win64_Shipping!ULevel::RouteActorInitialize() [C:\BuildAgent\work\b731a33f2a691e17\UE4\Engine\Source\Runtime\Engine\Private\Level.cpp:3345]
FactoryGameSteam_Engine_Win64_Shipping!UWorld::InitializeActorsForPlay() [C:\BuildAgent\work\b731a33f2a691e17\UE4\Engine\Source\Runtime\Engine\Private\World.cpp:5272]
FactoryGameSteam_Engine_Win64_Shipping!UEngine::LoadMap() [C:\BuildAgent\work\b731a33f2a691e17\UE4\Engine\Source\Runtime\Engine\Private\UnrealEngine.cpp:15338]
FactoryGameSteam_FactoryGame_Win64_Shipping!UFGGameEngine::LoadMap() [C:\BuildAgent\work\b731a33f2a691e17\UE4\FactoryGame\Source\FactoryGame\Private\FGGameEngine.cpp:31]
FactoryGameSteam_Engine_Win64_Shipping!UEngine::Browse() [C:\BuildAgent\work\b731a33f2a691e17\UE4\Engine\Source\Runtime\Engine\Private\UnrealEngine.cpp:14517]
FactoryGameSteam_Engine_Win64_Shipping!UEngine::TickWorldTravel() [C:\BuildAgent\work\b731a33f2a691e17\UE4\Engine\Source\Runtime\Engine\Private\UnrealEngine.cpp:14681]
FactoryGameSteam_Engine_Win64_Shipping!UGameEngine::Tick() [C:\BuildAgent\work\b731a33f2a691e17\UE4\Engine\Source\Runtime\Engine\Private\GameEngine.cpp:1788]
FactoryGameSteam_FactoryGame_Win64_Shipping!UFGGameEngine::Tick() [C:\BuildAgent\work\b731a33f2a691e17\UE4\FactoryGame\Source\FactoryGame\Private\FGGameEngine.cpp:15]
FactoryGameSteam_Win64_Shipping!FEngineLoop::Tick() [C:\BuildAgent\work\b731a33f2a691e17\UE4\Engine\Source\Runtime\Launch\Private\LaunchEngineLoop.cpp:5864]
FactoryGameSteam_Win64_Shipping!GuardedMain() [C:\BuildAgent\work\b731a33f2a691e17\UE4\Engine\Source\Runtime\Launch\Private\Launch.cpp:188]
FactoryGameSteam_Win64_Shipping!GuardedMainWrapper() [C:\BuildAgent\work\b731a33f2a691e17\UE4\Engine\Source\Runtime\Launch\Private\Windows\LaunchWindows.cpp:118]
FactoryGameSteam_Win64_Shipping!LaunchWindowsStartup() [C:\BuildAgent\work\b731a33f2a691e17\UE4\Engine\Source\Runtime\Launch\Private\Windows\LaunchWindows.cpp:258]
FactoryGameSteam_Win64_Shipping!WinMain() [C:\BuildAgent\work\b731a33f2a691e17\UE4\Engine\Source\Runtime\Launch\Private\Windows\LaunchWindows.cpp:298]
FactoryGameSteam_Win64_Shipping!__scrt_common_main_seh() [D:\a\_work\1\s\src\vctools\crt\vcstartup\src\startup\exe_common.inl:288]
kernel32
ntdll

### Expected Result
Game to load as normal :)
[Save Edit.zip](https://github.com/user-attachments/files/17270736/Save.Edit.zip)
[Personal Blueprint 2.zip](https://github.com/user-attachments/files/17270741/Personal.Blueprint.2.zip)
[Megaprint Global Rail Network 3.9 U8 (exp).zip](https://github.com/user-attachments/files/17270743/Megaprint.Global.Rail.Network.3.9.U8.exp.zip)

**Comment by AnthorNet** [comment]
Most likely won't fix the Drone issue without something that is easier to reproduce. Fils are simply too big here.

**Comment by caesarwease** [comment]
What Drone issue do you mean? I didn't mention any Drone.

**Comment by AnthorNet** [comment]
That is the error the crash report is stating, so most likely an error in Drone pasting.
I did try to copy/paste some without luck of reproducing the bug yet

**Comment by AnthorNet** [comment]
I've pasted both megaprints into a test save without any crash

**Comment by caesarwease** [comment]
By megaprints you’re including my own blueprint? And you pasted into an empty world - not the one I provided - and no crash?That’s very puzzling. I could try deleting the drones/ports to see if that helps. Maybe it’s bugging out somehow that the drones are in transit but not part of the blueprint, yet the port is expecting to see them? Doesn’t explain why you have no issue but maybe worth a try. On 8 Oct 2024, at 11:33, AnthorNet ***@***.***> wrote:﻿
I've pasted both megaprints into a test save without any crash
